lucien-ios/APIClient/LucienRequest.swift

```diff
@@ -17,6 +17,17 @@ enum LucienRequest: Request {
     case getCurrentUser
     case getDashboard
     case hasCollection
+    case createPhotoURL
+    case addComicBook(comicTitle: String,
```

Comment:
Instead of passing all of this in, pass a `Comic` and we can get the values from there